ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -1127,6 +1168,24 @@
 /* End PBXHeadersBuildPhase section */
 
 /* Begin PBXNativeTarget section */
+		60967E071D76B56D00863853 /* ADAL Preprocessing */ = {
```

Comment:
Apple does some of their own processing that we might be able to use, especially if we provide the build version as a build variable in an xcconfig file.